src/Physics/Resonance/XSection/BostedChristyEMPXSec.cxx

```diff
@@ -164,7 +164,8 @@ double BostedChristyEMPXSec::sigmaNR(int sf, double Q2, double W, bool isDeuteri
   
   double Wdif = W - (Mp + Mpi);
   
-  double m0 = 4.2802; // GeV
+  double m0 = sf==0?0.125:4.2802;
```

Comment:
Could you make this a bit easier to read by  adding some parentheses and spaces ala 
`double m0 = (sf==0) ?  0.125 : 4.2802;`
(also for the line below for `Q20`)
Also any reference for where these values come from

Reply:
Fixed